src/debian/rules

```diff
@@ -13,6 +13,12 @@ override_dh_auto_build:
 		$$python setup.py build; \
 	done;
 
+	sphinx-build -b html ../doc ./build/html
+	rm ./build/html/_static/jquery.js
```

Comment:
Why is this removing and linking important?

Reply:
Otherwise Lintian gives two warnings. I add a comment.

Reply:
I found a simpler way by using sphinxdoc debhelper ;)